### ISSUE TITLE
Feature request: select profiles/type/tags 

### ISSUE THREAD

**Issue by henryecw** [nickheyer/DiscoFlix#15]
functionality to select what quality profile, language profile, series type and tags are used on a show/movie

**Comment by nickheyer** [comment]
Somewhat of an expansion on #1 but I will keep it up since it's more feature heavy.

This is definitely in the works, but feature additions are slow without other contributors. Discord recently changed their username format and how it is processed with their API, so I have been using my available time to address that.

I will tag it as "help needed", hopefully someone assigns themselves to it quickly. Otherwise, I'll handle it when everything else is stable.

**Comment by henryecw** [comment]
Yeah I noticed it was pretty confusing setting up the config just now, because of the discord username update. Seemed to work eventually by adding `#0` to the end of the username when adding a user. Cool project by the way! Nice how active you are with it too

**Comment by nickheyer** [comment]
completed
